test/Visualization_Tests/src/Table_Spec.enso

```diff
@@ -54,7 +54,7 @@ visualization_spec connection =
 
             g = t.aggregate [Group_By "A", Group_By "B", Average "C"] . at "Average C"
             vis2 = Visualization.prepare_visualization g 1
-            json2 = make_json header=["Average C"] data=[[4]] all_rows=2 ixes_header=[] ixes=[]
+            json2 = make_json header=["Average C"] data=[[4.0]] all_rows=2 ixes_header=[] ixes=[]
```

Comment:
This is OK @jdunkerley @radeusgd ?

Reply:
Hmm, `Average` column should always have a Decimal type as an average of only integers can still not be an integer. So it looks good to me.

Reply:
This is result of changes in `HostToEnsoValue` or `Coersion*Node`. Now it chooses `double` variant, while previously it was using `fitIntoLong()` which was true for some `double` as well - I think.